cmd/kola/etcd.go

```diff
@@ -33,7 +33,7 @@ var cmdEtcd = &cli.Command{
 
 Work in progress: the code sets up an etcd cluster and then calls
 integration tests to see how etcd interacts with new CoreOS images.
-Currently discovery using another etcd cluster is supported for 0.4.7
+Currently discovery using an embedded etcd server is supported.
```

Comment:
This sentence doesn't parse for me. I think it is OK if "is supported" is removed and a comma is added after "Currently".